src/middleware/etag/index.test.ts

```diff
@@ -86,6 +86,7 @@ describe('Etag Middleware', () => {
     })
     res = await app.request(req)
     expect(res.status).toBe(304)
+    expect(res.headers.get('Etag')).toBe(etag)
```

Comment:
```suggestion
    expect(res.headers.get('ETag')).toBe(etag)
```

I know http headers are case-insensitive but just for consistency in the codebase.

Reply:
Wow, that's my mistake!